client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -102,6 +102,8 @@ public static FileSystem getHdfsFs(CelebornConf conf) {
       synchronized (ShuffleClient.class) {
         if (null == hdfsFs) {
           Configuration hdfsConfiguration = new Configuration();
+          // enable fs cache to avoid too many fs instances
+          hdfsConfiguration.set("fs.hdfs.impl.disable.cache", "false");
```

Comment:
Emm... does Hadoop `Configuration` support `setIfMissing` semantic like `SparkConf`?

Reply:
I'm concerning this hardcode, if no other options, please at least add a info log here.

Reply:
It is necessary to override cluster setting here.

Reply:
Logs added.